pkg/consts/controllers.go

```diff
@@ -51,6 +51,7 @@ const (
 	CtrlPodInternalNet         = "pod_internalnet"
 	CtrlPublicKey              = "publickey"
 	CtrlRouteConfiguration     = "routeconfiguration"
+	CtrlSecretWebhook          = "secret_webhook"
```

Comment:
Can you move this to the Core section?

Reply:
Yes, sorry, i focused on the alphabetical order and missed the group 😅